packages/@webex/plugin-meetings/test/unit/spec/meeting/locusMediaRequest.ts

```diff
@@ -414,6 +414,8 @@ describe('LocusMediaRequest.send()', () => {
 
     describe('confluence creation', () => {
       it('resolves without sending the request if LocalMute is requested before Roap Offer is sent (confluence state is "not created")', async () => {
+        assert.equal(locusMediaRequest.isConfluenceCreated(), false);
+
         const result = await sendLocalMute({audioMuted: false, videoMuted: true});
```

Comment:
In the `locusMediaRequest.send` method, can't we start using the new API instead of a string check? That way we keep all the checks for isConfluenceCreated pointed to one API. Avoids confusions.

Reply:
the check in `LocusMediaRequest.send()` is checking `this.confluenceState === 'not created'`. The `confluenceState` can have 3 different values `'not created' | 'creation in progress' | 'created'` so we can't use `!this.isConfluenceCreated()` there as this would be true for `'creation in progress'` value and in that case we want `LocusMediaRequest.send()` to queue the request and not resolve immediately.